src/@context/UserPreferences.tsx

```diff
@@ -77,6 +81,14 @@ function UserPreferencesProvider({
     localStorage?.allowExternalContent || false
   )
 
+  const [automationWallet, setAutomationWallet] = useState<string>(
+    localStorage?.automationWalletJSON || ''
+  )
+
+  const [automationWalletMode, setAutomationWalletMode] = useState<string>(
+    localStorage?.automationWalletMode || 'advanced'
```

Comment:
Only personal preference so far. I think the earlier iteration of the advanced mode was a bit better suited for demos

Maybe we should just move this to appConfig and allow configuration of this.

Reply:
I pointed this out because on the SBB portal we updated the onboarding module to show the simple view, so I thought it would make sense to have that as default to avoid confusing the user.

Reply:
Simply changed it to `SIMPLE` for now and added an issue to track the enhancement #440